### PR TITLE
KPI E

### PR DESCRIPTION
**Pull request by gaubert**
Pull request updating the temporal KPI created and ready for review

### CODE REVIEW

**Comment by tomkralidis** [comment]
@gaubert thanks for adding an example of maintenance frequency.

Should we add a case/point for identification status `onGoing`?  For example:

```xml
      <gmd:status>
        <gmd:MD_ProgressCode codeList="http://www.isotc211.org/2005/resources/Codelist/gmxCodelists.xml#MD_ProgressCode" codeSpace="ISOTC211/19115" codeListValue="onGoing">onGoing</gmd:MD_ProgressCode>
      </gmd:status>
```

I see a note from our [last meeting](https://github.com/wmo-im/wcmp/wiki/Meeting-2021-01-22#notes) about KPI E and going, but wanted to clarify if this is what we're looking for/adds value. Thoughts?

**Comment by tomkralidis** [comment]
Note that I've added dataset status in this branch for consideration.  If we don't agree on Friday we can remove.

File: KPI/E.adoc [hunk 1]
```diff
@@ -1,106 +1,147 @@
...
+        </gml:TimePeriod>
+      </gmd:extent>
+    </gmd:EX_TemporalExtent>
+  </gmd:temporalElement>
+  ...
+</gmd:identificationInfo>
+```
+
+In the case of an ongoing dataset with a known start date, but no known end
+date can be defined, the end date much be defined as `now`.  For instance,
```
Comment (josusky) [inline KPI/E.adoc:49]: There seems to be a typo "the end date **much** be defined ", I am not sure if you meant "must" or "may" :-)

File: KPI/E.adoc [hunk 2]
```diff
@@ -1,106 +1,147 @@
...
+         </gml:TimePeriod>
+       </gmd:extent>
+     </gmd:EX_TemporalExtent>
+  </gmd:temporalElement>
+</gmd:temportalElement>
+```
+
+The Data Update Frequency is defined in `gmd:maintenanceAndUpdateFrequency`:
+
+Below is an example of a GTS bulletin whic is updated every 6 hours.
```
Comment (josusky) [inline KPI/E.adoc:70]: One more typo "bulletin **whic**".

File: KPI/E.adoc [hunk 3]
```diff
@@ -1,106 +1,147 @@
...
+|The Temporal Extent contains begin and end components
+|1
+
+|The begin date time is less than or equal to the end date time
+|1
+
+|The Data Update Frequency is defined
+|1
+
+|The Data Status is defined
```
Comment (josusky) [inline KPI/E.adoc:139]: Is the "gmd:status" obligatory or is this only to motivate people to use it?

File: KPI/E.adoc (hunk 1 above)
Comment (gaubert) [reply KPI/E.adoc]: yep thanks. done

File: KPI/E.adoc (hunk 2 above)
Comment (gaubert) [reply KPI/E.adoc]: same updated here

**Comment by gaubert** [comment]
Updates as discussed during the meeting done  :-)

File: KPI/E.adoc (hunk 3 above)
Comment (tomkralidis) [reply KPI/E.adoc:139]: as discussed, not obligatory.

**Pull request merged**